js/gated-content/src/components/personal-training/PersonalTrainingTeaser.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="teaser event-teaser personal-training-teaser">
+    <router-link
+      :to="{ name: 'PersonalTraining', params: { tid: video.id } }"
+    >
+      <div class="title">{{ video.attributes.title }}</div>
```

Comment:
Very minor. We can rename `video` to personal_training for better readability

Reply:
The use of an `EventMixin` will not allow us to have a different param name than a `video`